### PR TITLE
Fix typos in tutorial 4-2

### PR DESCRIPTION
**Pull request by morinokami**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)

#### Description

- Fix typos in tutorial 4-2.

### CODE REVIEW

File: src/content/docs/en/tutorial/4-layouts/2.mdx [hunk 1]
```diff
@@ -74,7 +74,7 @@ Here's an example of refactoring your code to include the `pubDate` in the layou
 ```markdown title="src/pages/posts/post-1.md" del={1}
 Published on: 2022-07-01
 
-Welcome to my new blog about learning Astro! Here, I will share my learning journey as I build a new website.
+Welcome to my _new blog_ about learning Astro! Here, I will share my learning journey as I build a new website.
```
Comment (morinokami) [inline src/content/docs/en/tutorial/4-layouts/2.mdx:77]: This line was added in Tutorial 2-2, and "my blog" should be italicized. 

https://github.com/withastro/docs/blob/8fbd915f1a6ac3da4d827490655fccc431ce22f8/src/content/docs/en/tutorial/2-pages/2.mdx#L58

File: src/content/docs/en/tutorial/4-layouts/2.mdx [hunk 2]
```diff
@@ -131,14 +131,14 @@ Can you figure out what should go in the blanks so that the following two compon
...
-    <p>Written by:{frontmatter.______ } on {frontmatter.pubDate}</p>
+    <p>Written by: {frontmatter.______ } on {frontmatter.pubDate}</p>
     < _______ />
     <Footer />
     ```
     <details>
         <summary>Show the blanks filled in!</summary>
 
-        1.  ```markdown title="src/pages/posts/learning-astro.md" "../layouts" "pubDate:"
+        1.  ```markdown title="src/pages/posts/learning-astro.md" "layouts" "pubDate:"
```
Comment (morinokami) [inline src/content/docs/en/tutorial/4-layouts/2.mdx:141]: `../` is already included, so it shouldn't be highlighted.

Screenshot 2023-07-16 at 17 04 18

Comment (EEP-Benny) [reply src/content/docs/en/tutorial/4-layouts/2.mdx:141]: Haha, I was about to make the same changes 😄 Now I don't have to anymore.

Can you also remove the `:` after `pubDate` (and the `.` before `author` below) from being highlighted?

I also noticed that the filename is `MarkdownLayout.astro`, but it is referenced as `MyMarkdownLayout.astro`. Does that fit into your PR as well?

Comment (morinokami) [reply src/content/docs/en/tutorial/4-layouts/2.mdx]: Hey @EEP-Benny, thanks for your review! I have corrected the lines you pointed out, so could you please check again?

File: src/content/docs/en/tutorial/4-layouts/2.mdx [hunk 3]
```diff
@@ -148,13 +148,13 @@ Can you figure out what should go in the blanks so that the following two compon
...
 
-        2.  ```astro title="src/layouts/MarkdownLayout.astro" " Footer " " frontmatter " ".author" "slot"
+        2.  ```astro title="src/layouts/MyMarkdownLayout.astro" " Footer " " frontmatter " "author" "slot"
             ---
             import Footer from '../components/Footer.astro'
             const { frontmatter } = Astro.props
             ---
             <h1>{frontmatter.title}</h1>
-            <p>Written by:{frontmatter.author } on {frontmatter.pubDate}</p>
+            <p>Written by: {frontmatter.author } on {frontmatter.pubDate}</p>
```
Comment (EEP-Benny) [inline src/content/docs/en/tutorial/4-layouts/2.mdx:157]:
```suggestion
            <p>Written by: {frontmatter.author} on {frontmatter.pubDate}</p>
```
Same thing above (with the blanks).

**Review by EEP-Benny — APPROVED** [review]
Exactly how I would have changed it 👍 
But I found another small thing just now, which you might want to fix in this PR as well.

**Comment by morinokami** [comment]
@EEP-Benny Ah, good catch! I also have fixed another one in https://github.com/withastro/docs/pull/3732/commits/9c82b7d11a75ca79ab9850b961ce746d6aad6e69. Hopefully this is the last one😅

**Review by Yan-Thomas — APPROVED** [review]
Nice catch peeps, thanks for flagging all these typos!

**Pull request merged**